PIM/Integration/03_ManageAttributeGroups.md

```diff
@@ -0,0 +1,263 @@
+
+2. Click and hold the ![Sort](../../Assets/Icons/Sort01.png "[Sort]") (Sort) button to the left of the attribute group you want to move to another position in the list.
+
+3. By using drag and drop, move the selected attribute group to the desired position in the list.
+
+4. Click the [SAVE] button in the upper right corner.  
+  The new attribute group order has been saved. The *Edit Attribute Groups* view is closed.
+
+
+
+### Delete an attribute group
+
+You can delete an attribute group if it is no longer in use. Note, that you just delete the attribute group but not the attributes within a group. All active attributes that are no longer assigned to an attribute group are automatically assigned to a new attribute group with the name *Unassigned Group* which is only displayed in the product view. If you delete an attribute group with sub-groups, also the attribute sub-groups are deleted. The procedure to delete attribute groups and sub-groups is identical. Just select the attribute sub-group instead of the group and follow the steps described below.
+
+#### Prerequisites
+
+At least one attribute group has been created, see [Create an attribute group](#create-an-attribute-group).
+
+#### Procedure
+*PIM > Settings > Tab ATTRIBUTE GROUPS*
+
+![Attribute groups](../../Assets/Screenshots/PIM/Settings/AttributeGroups/AttributeGroups.png "[Attribute groups]")
+
+1. Click the ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit) button to the right of the *Attribute Groups* section in the left column.   
+  The *Edit Attribute Groups* view is displayed.
+
+  ![Edit attribute groups](../../Assets/Screenshots/PIM/Settings/AttributeGroups/EditAttributeGroups.png "[Edit attribute groups]")
+
+2. Click the ![Edit](../../Assets/Icons/Edit03.png "[Edit]") (Edit) button to the right of the attribute group you want to delete.   
+  The attribute group data is displayed in a window in the attribute group column.
+
+  ![Edit attribute group data](../../Assets/Screenshots/PIM/Settings/AttributeGroups/EditAttributeGroupData.png "[Edit attribute group data]")
+
+3. Click the ![Delete](../../Assets/Icons/Trash01.png "[Delete]") (Delete) button in the bottom left corner of the attribute group data window.    
+  The selected attribute group has been removed from the attribute group column. The attribute group data window is closed.
+
+4. Click the [SAVE] button in the upper right corner.  
+  The attribute group has been deleted. The *Edit Attribute Groups* view is closed.
+
+
+
+### Add attributes to a group
+
+You can add attributes to an attribute group or sub-group. Note that you can add an attribute only to an attribute group without sub-groups. Otherwise, you have to add the attribute to an attribute sub-group. Be aware, that an attribute can be assigned to one attribute group or sub-group only. When you add an attribute to an attribute group that is already added to another group, it is automatically removed from the former attribute group.
```

Comment:
```suggestion
You can add attributes to an attribute group or sub-group. Note that you can add an attribute only to an attribute group without sub-groups. Otherwise, you have to add the attribute to an attribute sub-group. Be aware that an attribute can be assigned to one attribute group or sub-group only. When you add an attribute to an attribute group that is already added to another group, it is automatically removed from the former attribute group.
```